### ISSUE TITLE
Configure parameters

### ISSUE THREAD

**Issue by FoilHao** [yangxue0827/R2CNN_FPN_Tensorflow#70]
When you say configuring parameters in $R2CNN_ROOT/libs/configs/cfgs.py, do I really need to adjust the parameters? I mean, if they are well set, I might just leave them.
I am a bit new in this area and working on detecting different kinds of bills and things like driving licenses, ID cards, bank cards and so on. 
I wonder if you can give me some advice on parameters, thank you in advance!

**Comment by yangxue0827** [comment]
You mat need to modify ANCHOR_SCALES, ANCHOR_RATIOS, LR, DECAY_STEP and so on， it is determined by the characteristics of your data set. 
PS: I recommend using this [repo](https://github.com/DetectionTeamUCAS/R2CNN_Faster-RCNN_Tensorflow), which has better performance.

**Comment by FoilHao** [comment]
Thanks very much for your advice

**Comment by FoilHao** [comment]
Meanwhile ,I wonder if this model works fine on detecting things like tickets, ID cards, bank cards,driving licenses and son on.

**Comment by yangxue0827** [comment]
I have not tried these, so I am not quite sure. @FoilHao

**Comment by FoilHao** [comment]
Thank you, by the way, where should I put the test images before running test1.py?

**Comment by FoilHao** [comment]
I just tested with my own dataset and it worked pretty good!
Thanks so much for your repo and it's super amazing!
But I have one more question, I mean, I have had so many these days:) (Thanks for your patience)
I used the VOCtestset(including images and annotations) to generate a tfrecord file, which was used in test1.py to output processed images, and the results look great.
But in real situation, we usually have some images and want the model to directly output detection results from them. We don't want to label and convert them to tfrecords. So can I achieve this funtion(input images, run some codes and get results) through your work?
Thanks so much!!!(This is probably my final question...:)
